_security/access-control/permissions.md

```diff
@@ -134,37 +172,117 @@ These permissions are for the cluster and can't be applied granularly. For examp
 - cluster:admin/opendistro/reports/instance/list
 - cluster:admin/opendistro/reports/instance/get
 - cluster:admin/opendistro/reports/menu/download
+
+### Transform job permissions
+
+See [Transforms APIs]({{site.url}}{{site.baseurl}}/im-plugin/index-transforms/transforms-apis/)
+
 - cluster:admin/opendistro/transform/index
 - cluster:admin/opendistro/transform/get
 - cluster:admin/opendistro/transform/preview
 - cluster:admin/opendistro/transform/delete
 - cluster:admin/opendistro/transform/start
 - cluster:admin/opendistro/transform/stop
 - cluster:admin/opendistro/transform/explain
+
+### Observability permissions
+
+See [Observability security]({{site.url}}{{site.baseurl}}/observing-your-data/observability-security/).
+
 - cluster:admin/opensearch/observability/create
 - cluster:admin/opensearch/observability/update
 - cluster:admin/opensearch/observability/delete
 - cluster:admin/opensearch/observability/get
+
+### Cross-cluster replication
+
+See [Cross-cluster replication security]({{site.url}}{{site.baseurl}}/tuning-your-cluster/replication-plugin/permissions/).
+
 - cluster:admin/plugins/replication/autofollow/update
+
+### Reindex
+
+See [Reindex document]({{site.url}}{{site.baseurl}}/api-reference/document-apis/reindex/).
+
 - cluster:admin/reindex/rethrottle
+
+### Snapshot repository permissions
+
+See [Snapshot APIs]({{site.url}}{{site.baseurl}}/api-reference/snapshots/index/).
+
 - cluster:admin/repository/delete
 - cluster:admin/repository/get
 - cluster:admin/repository/put
 - cluster:admin/repository/verify
+
+### Reroute
+
+See [Cluster manager task throttling]({{site.url}}{{site.baseurl}}/tuning-your-cluster/cluster-manager-task-throttling/).
+
 - cluster:admin/reroute
+
+### Script permissions
+
+See [Script APIs]({{site.url}}{{site.baseurl}}/api-reference/script-apis/index/).
+
 - cluster:admin/script/delete
 - cluster:admin/script/get
 - cluster:admin/script/put
+
+### Update settings permission
+
+See [Update settings]({{site.url}}{{site.baseurl}}api-reference/index-apis/update-settings/) in Index APIs.
+
 - cluster:admin/settings/update
+
+### Snapshot permissions
+
+See [Snapshot APIs]({{site.url}}{{site.baseurl}}/api-reference/snapshots/index/).
+
 - cluster:admin/snapshot/create
 - cluster:admin/snapshot/delete
 - cluster:admin/snapshot/get
 - cluster:admin/snapshot/restore
 - cluster:admin/snapshot/status
 - cluster:admin/snapshot/status*
+
+### Task permissions
+
+See [Tasks]({{site.url}}{{site.baseurl}}/api-reference/tasks/) APIs.
```

Comment:
Should APIs be part of the hyperlinked text, like preceding sections?

Reply:
Just highlighting the text from the level 1 heading of the topic.